content/docs/gke/private-clusters.md

```diff
@@ -24,6 +46,165 @@ Before installing Kubeflow ensure you have installed the following tools:
   * [gcloud](https://cloud.google.com/sdk/)
   * [jsonnet](https://github.com/google/jsonnet/releases)
 
+
+You will need to know your gcloud organization id and project number; you can get it via gcloud
+
+```
+export ORGANIZATION=$(gcloud organizations list --filter=DISPLAY_NAME=<ORG_NAME> --format='value(name)')
+export PROJECT_NUMBER=$(gcloud projects describe kubeflow-dev --format='value(projectNumber)')
+```
+
+## Known issues
+
+* [kubeflow/kubeflow#3106](https://github.com/kubeflow/kubeflow/issues/3106) cloud endpoints ksonnet component doesn't use the override parameter for the image.
+
+
+## Enable VPC Service Controls In Your Project
```

Comment:
We should use the same order for the 2 options (VPC Service Controls vs private GKE) everywhere we mention them on the page. For example, this section comes before the private GKE section, but in the intro private GKE comes first.

Reply:
Done. Flipped the order so mention VPC service controls first.